arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -243,6 +243,14 @@ config ZERO_LATENCY_IRQS
 	  higher priority than the rest of the kernel they cannot use any
 	  kernel functionality.
 
+config DYNAMIC_DIRECT_INTERRUPTS
+	bool "Enable support for dynamic direct interrupts"
+	select DYNAMIC_INTERRUPTS
+	help
+	  Enable installation of interrupt service routines for Direct
+	  interrupts at runtime.
+	  Note: this enables support for dynamic interrupts in the kernel.
```

Comment:
Could flesh out this help text a bit maybe. Could explain what direct interrupts are (and maybe lowercase 'direct'), give some links, etc. In the Linux kernel, `checkpatch.pl` warns if the help text is less than four lines, but maybe our version is old.

Reply:
I see. So, I've changed 'Direct' to lowercase, thanks.

Well, the provided API (see in this PR) has all the documentation for direct dynamic interrupts. So I've just copied here a minor phrase briefly explaining direct interrupts.

In general the help-text of Kconfig symbols does not need to explain in detail the feature/functionality; we've got documentation for that. :)